index.html

```diff
@@ -0,0 +1,159 @@
+       <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+       <ul class="skils">
+        <li>html</li>
+        <li class="skil">bootstrap</li>
+        <li class="skil">Ruby</li>
+     </ul>
+     </div>
+     <a class="see-project-2" href="">See Project</a>
+
+
+     <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li>html</li>
+       <li class="skil">bootstrap</li>
+       <li class="skil">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2" href="">See Project</a>
+    
+</section>
+
+<section class="section-3">
+   <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31">Hello I’m a software developer! I can help you build a product ,feature or 
+         website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a class="get-myresume" href="">Get My Resume</a>
```

Comment:
As the previous reviewer requested, kindly fix the padding of the "get my resume" button.
<img width="418" alt="Screenshot 2023-01-19 at 10 58 13 PM" src="https://user-images.githubusercontent.com/25887253/213549021-22e775f3-aef8-4d23-b027-69833ac32337.png">
<img width="974" alt="Screenshot 2023-01-19 at 10 59 18 PM" src="https://user-images.githubusercontent.com/25887253/213549024-ed14cea9-018b-43e2-b12d-8695a396f5d4.png">

[OPTIONAL] Please add hover and active states to all of your button like on the template.
<img width="499" alt="Screenshot 2023-01-19 at 11 03 49 PM" src="https://user-images.githubusercontent.com/25887253/213549249-1dd5cfad-b818-41d5-82b0-13ac156df264.png">
<img width="499" alt="Screenshot 2023-01-19 at 11 04 04 PM" src="https://user-images.githubusercontent.com/25887253/213549252-5b41ed72-1a97-4bed-aec9-a25f239c05b6.png">